socorro/cron/jobs/bugzilla.py

```diff
@@ -184,9 +180,15 @@ def inner_transaction(
         if not useful:
             raise NothingUsefulHappened('nothing useful done')
 
-    def _iterator(self, query):
-        for report in csv.DictReader(urllib2.urlopen(query)):
+    def _iterator(self, from_date):
+        payload = BUGZILLA_PARAMS.copy()
+        payload['chfieldfrom'] = from_date
+        r = requests.get(BUGZILLA_BASE_URL, params=payload)
+        if r.status_code < 200 or r.status_code >= 300:
+            raise BugzillaConnectionError()
```

Comment:
This should have some kind of message with the status code any anything else that might be helpful in figuring out why it failed to connect.